README.md

```diff
@@ -1,2 +1,25 @@
-# hw01
-Repository for homework 01
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# HOMEWORK 1
+
+__*Hello*__
+
+My name is Elaine Chen. I'm a third-year undergraduate student at the University of Chicago studying Political Science and Economics.
+
+![Portrait](https://raw.githubusercontent.com/elaineywchen/hw01/master/portrait%20for%20r.jpg){width=250px}
+
+##Where I'm from
```

Comment:
There is a mistake in the formatting. In order to successfully add a header, I think you need to do this: # blah blah blah. But also, it will be helpful to view the output in order to notice any mistakes.